apis/v1alpha1/shared_types.go

```diff
@@ -98,19 +84,58 @@ type BpfAppCommon struct {
 	// Bytecode configures where the bpf program's bytecode should be loaded
 	// from.
 	ByteCode BytecodeSelector `json:"bytecode"`
+
+	// MapOwnerSelector is used to select the loaded eBPF program this eBPF program
+	// will share a map with. The value is a label applied to the BpfProgram to select.
+	// The selector must resolve to exactly one instance of a BpfProgram on a given node
+	// or the eBPF program will not load.
```

Comment:
we no longer have an object called `BpfProgram` so we should avoid using it everywhere in code and comments

Reply:
I got rid of a lot of it, but there's still more work to do.